### PR TITLE
[package] error if we try to mock a module in 3.6

### PR DESCRIPTION
**Pull request by suo**
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#61469**

This feature is not supported, error out early.

Differential Revision: [D29639797](https://our.internmc.facebook.com/intern/diff/D29639797)

### CODE REVIEW

**Comment by suo** [comment]
@suo has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D29639797).

File: torch/package/package_exporter.py [hunk 1]
```diff
@@ -706,6 +707,11 @@ def mock(
                 If ``allow_empty=True``, no such exception is thrown.
 
         """
+        if version_info < (3, 6):
```
Comment (Lilyjjo) [inline torch/package/package_exporter.py:710]: should this be <= or (3,7)?

Comment (suo) [reply torch/package/package_exporter.py:710]: yes! DAMMIT github copilot failed me

**Comment by suo** [comment]
@suo has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D29639797).

**Review by lobstergrindset — APPROVED** [review]

**Pull request closed without merging**